### ISSUE TITLE
Heroin dependence

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6533]
**Preferred term label:** Heroin dependence

**Synonyms**
Heroin addiction 

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Addictive behavior HP:0030858

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
adding but with parent Opioid addiction
